tensorboard/components/tensor_widget/dtype-helper-test.ts

```diff
@@ -0,0 +1,56 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+import {isIntegerDType, isFloatDType} from './dtype-helper';
+
+describe('isIntegerDType', () => {
+  it('unsigned ints', () => {
+    expect(isIntegerDType('uint8')).toEqual(true);
```

Comment:
chai is supposed to be fluent. Please prefer https://www.chaijs.com/api/bdd/#true

Reply:
Separately, `to.equal` is the pattern we use.